go/cmd/dolt/dolt.go

```diff
@@ -136,7 +136,7 @@ func runMain() int {
 	restoreIO := cli.InitIO()
 	defer restoreIO()
 
-	warnIfMaxFilesTooLow()
```

Comment:
Why are we commenting this out?

Reply:
For some reason this fail in GoLand for me so I comment it out locally. Reverted.